scripts/setup/nrfconnect/get_zap.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# The script parses integrations/docker/images/chip-build/Dockerfile file
+# from the Matter repository and looks for ENV ZAP_VERSION= string to find
+# currently recommended ZAP version. After that the package matching this version
+# and currently used OS is downloaded from the ZAP release packages and extracted
+# in the location given by the user.
+
+import os
+import stat
+import re
+import wget
```

Comment:
The `wget` python library isn't the default one. Is it added to any `requirements.txt` file?

Reply:
No, I was going to add it to the NCS requirements.txt

Reply:
Added in related sdk-nrf PR.